src/main/java/com/telefonica/iot/cygnus/sinks/OrionSink.java

```diff
@@ -733,29 +733,16 @@ public void initialize(long startDateMs) {
             accTransactionIds = "";
         } // initialize
         
-        /**
-         * Gets a copy of this accumulator, except for the already persisted sub-batches.
-         * @return A copy of this accumulator, except for the already persisted sub-batches
-         */
-        public Accumulator getAccumulatorForRollback() {
-            Accumulator accumulatorForRollback = new Accumulator();
-            accumulatorForRollback.accIndex = this.accIndex;
-            accumulatorForRollback.accStartDate = this.accStartDate;
-            accumulatorForRollback.accTransactionIds = this.accTransactionIds;
-            Set<String> destinations = this.batch.getDestinations();
-            
-            for (String destination : destinations) {
-                if (!this.batch.isPersisted(destination)) {
-                    ArrayList<CygnusEvent> events = this.batch.getEvents(destination);
-                    
-                    for (CygnusEvent event : events) {
-                        accumulatorForRollback.batch.addEvent(destination, event);
-                    } // for
-                } // if
-            } // for
-
-            return accumulatorForRollback;
-        } // getAccumulatorForRollback
+        @Override
+        public Accumulator clone() {
+            try {
+                Accumulator acc = (Accumulator) super.clone();
+                return acc; 
+            }
+            catch (CloneNotSupportedException ce) {
```

Comment:
`catch` must be after the `{`. i.e.:

    try {
       ...
    } catch {
       ...
    }